[2024-05-21T18:59:01.459+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-21T18:59:01.474+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: games.create_db manual__2024-05-21T18:58:58.841098+00:00 [queued]>
[2024-05-21T18:59:01.478+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: games.create_db manual__2024-05-21T18:58:58.841098+00:00 [queued]>
[2024-05-21T18:59:01.478+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-05-21T18:59:01.487+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): create_db> on 2024-05-21 18:58:58.841098+00:00
[2024-05-21T18:59:01.495+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=64) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-21T18:59:01.496+0000] {standard_task_runner.py:63} INFO - Started process 67 to run task
[2024-05-21T18:59:01.496+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'games', 'create_db', 'manual__2024-05-21T18:58:58.841098+00:00', '--job-id', '555', '--raw', '--subdir', 'DAGS_FOLDER/get_main_games.py', '--cfg-path', '/tmp/tmpet03ai25']
[2024-05-21T18:59:01.497+0000] {standard_task_runner.py:91} INFO - Job 555: Subtask create_db
[2024-05-21T18:59:01.526+0000] {task_command.py:426} INFO - Running <TaskInstance: games.create_db manual__2024-05-21T18:58:58.841098+00:00 [running]> on host 70309da0b19a
[2024-05-21T18:59:01.580+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='games' AIRFLOW_CTX_TASK_ID='create_db' AIRFLOW_CTX_EXECUTION_DATE='2024-05-21T18:58:58.841098+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-21T18:58:58.841098+00:00'
[2024-05-21T18:59:01.580+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-21T18:59:01.600+0000] {base.py:84} INFO - Using connection ID '***' for task execution.
[2024-05-21T18:59:02.893+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-21T18:59:02.893+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-21T18:59:02.900+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=games, task_id=create_db, execution_date=20240521T185858, start_date=20240521T185901, end_date=20240521T185902
[2024-05-21T18:59:02.951+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-21T18:59:02.970+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-21T18:59:02.971+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
